### DISCUSSION TITLE
maximum recursion depth exceeded while calling a Python object

### DISCUSSION THREAD

**Discussion by shailesh** [pallupz/covid-vaccine-booking#158] [General]
maximum recursion depth exceeded while calling a Python object
while getting captcha

================================= GETTING CAPTCHA ==================================================
Captcha Response Code: 200
x_order_2: colinear!
x_order_2: colinear!
x_order_2: colinear!
x_order_2: colinear!
x_order_2: colinear!
x_order_2: colinear!
x_order_2: colinear!
x_order_2: colinear!
x_order_2: colinear!
colinear!
colinear!
x_order_2: colinear!
x_order_2: colinear!
x_order_2: colinear!
maximum recursion depth exceeded while calling a Python object
===================================================================================
Centers available in Dumka from 16-05-2021 as of 2021-05-16 15:49:57: 48
+-------+---------------------+------------+-----------+-------------+------------+------------------------------------------------------------------------------+
|   idx | name                | district   |   pincode |   available | d

**Comment by ashish0936** [comment]
yes i am getting the same error

**Reply by ashish0936** [reply]
if you find some solution. Let me know please
